docker-compose.oci8.yaml

```diff
@@ -1,7 +0,0 @@
-#ddev-generated
-services:
-  web:
-    # The platform statement here has no effect on AMD64/X86 systems
-    # On ARM64 (macOS Apple Silicon) it will make it work if Rosetta2 is working. The entire
-    # web container becomes amd64.
-    platform: linux/amd64
```

Comment:
Please don't remove this critical file. Not sure what happened here.

Reply:
@rfay, is it required to add this for the current setup, as it already supports AMD and ARM processors?